src/app/workflow/tool-tab/tool-tab.service.ts

```diff
@@ -46,6 +48,8 @@ export class ToolTabService {
         return 'task\xa0ID';
       case ToolDescriptor.TypeEnum.NFL:
         return 'process\xa0name';
+      case ToolDescriptor.TypeEnum.GXFORMAT2:
+        return 'tool\xa0ID';
```

Comment:
Comment on weird escape.

Reply:
FYI, was copied from CWL case. 
Discussed in person whether it is more appropriate to call this steps or tools (they seem to be used interchangeably looking together)